File: Microservicios/backend/src/main/java/com/P/Microservicios/Model/Persona.java
package com.P.Microservicios.Model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Persona {
    private Long id;
    private String nombre;
    private String direccion;
    private String telefono;
    private String correo;
}

File: Microservicios/frontend/models/persona.js
export class Persona {
    constructor({ id = null, nombre = "", direccion = "", telefono = "", correo = "" } = {}) {
        this.id = id;
        this.nombre = nombre;
        this.direccion = direccion;
        this.telefono = telefono;
        this.correo = correo;
    }

    toJSON() {
        return {
            id: this.id,
            nombre: this.nombre,
            direccion: this.direccion,
            telefono: this.telefono,
            correo: this.correo,
        };
    }
}

